--- cliff-2.18.0/cliff/tests/test_help.py.orig	2024-01-16 20:13:24.252285303 +0100
+++ cliff-2.18.0/cliff/tests/test_help.py	2024-01-16 20:49:44.113809312 +0100
@@ -107,7 +107,7 @@ class TestHelp(base.TestBase):
         help_text = stdout.getvalue()
         basecommand = os.path.split(sys.argv[0])[1]
         self.assertIn('usage: %s [--version]' % basecommand, help_text)
-        self.assertIn('optional arguments:\n  --version', help_text)
+        self.assertRegex(help_text, 'option(s|al arguments):\n  --version', help_text)
         expected = (
             '  one            Test command.\n'
             '  three word command  Test command.\n'
