--- cliff-2.18.0/cliff/tests/test_app.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_app.py	2024-01-16 20:07:52.707414769 +0100
@@ -21,7 +21,10 @@ except ImportError:
     from io import StringIO
 import sys
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 import six
 
 from cliff import app as application
--- cliff-2.18.0/cliff/tests/test_command_hooks.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_command_hooks.py	2024-01-16 20:08:13.197303765 +0100
@@ -18,7 +18,10 @@ from cliff import lister
 from cliff import show
 from cliff.tests import base
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 from stevedore import extension
 
 
--- cliff-2.18.0/cliff/tests/test_commandmanager.py.orig	2024-01-16 20:08:35.477183065 +0100
+++ cliff-2.18.0/cliff/tests/test_commandmanager.py	2024-01-16 20:08:38.507166650 +0100
@@ -10,7 +10,10 @@
 #  License for the specific language governing permissions and limitations
 #  under the License.
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 import testscenarios
 
 from cliff import command
--- cliff-2.18.0/cliff/tests/test_complete.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_complete.py	2024-01-16 20:09:30.896882831 +0100
@@ -13,7 +13,10 @@
 """Bash completion tests
 """
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 
 from cliff import app as application
 from cliff import commandmanager
--- cliff-2.18.0/cliff/tests/test_formatters_csv.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_formatters_csv.py	2024-01-16 20:09:46.763463541 +0100
@@ -16,7 +16,10 @@
 import argparse
 import unittest
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 import six
 
 from cliff.formatters import commaseparated
--- cliff-2.18.0/cliff/tests/test_formatters_json.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_formatters_json.py	2024-01-16 20:10:12.289991918 +0100
@@ -18,7 +18,10 @@ from cliff.formatters import json_format
 from cliff.tests import base
 from cliff.tests import test_columns
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 import six
 
 
--- cliff-2.18.0/cliff/tests/test_formatters_shell.py.orig	2024-01-16 20:10:30.733225336 +0100
+++ cliff-2.18.0/cliff/tests/test_formatters_shell.py	2024-01-16 20:10:36.759859354 +0100
@@ -19,7 +19,10 @@ from cliff.formatters import shell
 from cliff.tests import base
 from cliff.tests import test_columns
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 
 
 class TestShellFormatter(base.TestBase):
--- cliff-2.18.0/cliff/tests/test_formatters_table.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_formatters_table.py	2024-01-16 20:10:55.843089304 +0100
@@ -16,7 +16,10 @@ import argparse
 import os
 import textwrap
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 from six import StringIO
 
 from cliff.formatters import table
--- cliff-2.18.0/cliff/tests/test_formatters_yaml.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_formatters_yaml.py	2024-01-16 20:11:10.639675811 +0100
@@ -19,7 +19,10 @@ from cliff.formatters import yaml_format
 from cliff.tests import base
 from cliff.tests import test_columns
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 
 
 class TestYAMLFormatter(base.TestBase):
--- cliff-2.18.0/cliff/tests/test_help.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_help.py	2024-01-16 20:11:20.262957010 +0100
@@ -17,7 +17,10 @@ except ImportError:
 import os
 import sys
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 
 from cliff import app as application
 from cliff import commandmanager
--- cliff-2.18.0/cliff/tests/test_lister.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_lister.py	2024-01-16 20:11:28.762910962 +0100
@@ -17,7 +17,10 @@ import weakref
 from cliff import lister
 from cliff.tests import base
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 
 
 class FauxFormatter(object):
--- cliff-2.18.0/cliff/tests/test_show.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_show.py	2024-01-16 20:11:54.516104778 +0100
@@ -17,7 +17,10 @@ import weakref
 from cliff import show
 from cliff.tests import base
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 
 
 class FauxFormatter(object):
--- cliff-2.18.0/cliff/tests/test_utils.py.orig	2020-01-08 21:36:15.000000000 +0100
+++ cliff-2.18.0/cliff/tests/test_utils.py	2024-01-16 20:12:03.082725036 +0100
@@ -17,7 +17,10 @@ import struct
 import sys
 import unittest
 
-import mock
+try:
+    from unittest import mock
+except ImportError:
+    import mock
 
 from cliff import utils
 from cliff.tests import base
